README.md

```diff
@@ -363,19 +353,142 @@ https://github.com/dwyl/learn-codeclimate
 <br /> <br /> <br /> <br />
 
 
-## tl;dr
+# tl;dr
 
 There is _no shortage_ of options available for
-Technology Stack!
+Technology Stack! <br />
 See: https://www.google.com/search?q=technology+stack&tbm=isch <br />
-How did we _arrive_ at the conclusion that "PETE"
+So, _how_ did we _arrive_ at the conclusion that "PETE"
 was "_the **one**_" for us...?
+We _already_ had a _really_ good
+[Node.js Stack](https://github.com/dwyl/technology-stack#nodejs-stack)
+which worked well for us and our clients. so ...
+
+
+## Why Try a "New Stack"?
+
+#### Why Try Something New When We're _Already_ Good with the "Old"...?
+
+Our _reasoning_ for
+_considering_ an alternative approach/stack for building web apps
+was fueled by our
+[_curiousity_](https://www.goodreads.com/quotes/tag/curiosity)
+and
+["_shoshin_"](https://en.wikipedia.org/wiki/Shoshin). <br />
+
+> "_The important thing is not to **stop questioning**. **Curiosity** has its own reason for existence._" ~ Albert Einstein <br /> ~
+"Old Man's Advice to Youth: '**Never Lose** a Holy ***Curiosity***.'"
+LIFE Magazine (2 May 1955) p. 64”
+
+In November 2016 we (_once again_) **questioned our _assumptions_**,
+***re-examined*** and
+[***surveyed***](https://github.com/dwyl/learn-elm/issues/10)
+the "landscape" of "_emerging trends_" in web app development.
+We were ~~pleasantly surprised~~ ***delighted*** to see the _amazing progress_
+made by the people in the Elixir, Phoenix and Elm communities!
+These technologies are set to "***take off***" in 2017 and we are _excited_
+to be _sharing_ the technical/competitve advantage with our clients!
 
 Please see: https://github.com/dwyl/learn-phoenix-framework#questions
 
```

Comment:
Do we want to add something to this section around switching our stack but still using the right tool for the job rather than having a 'blanket stack'?